<ui version="4.0" >
 <class>GameRoomPrepareClass</class>
 <widget class="QWidget" name="GameRoomPrepareClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="bFight" >
       <property name="text" >
        <string>В бой</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QPushButton" name="bExit" >
       <property name="text" >
        <string>Выход</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Выбор техники</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="1" column="2" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Статистика</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="lInFight" >
          <property name="text" >
           <string>В бою:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lAll" >
          <property name="text" >
           <string>Всего:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QComboBox" name="cbTank" >
       <item>
        <property name="text" >
         <string>СТ</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>ТТ</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lState" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
